integration-tests/jpa-postgresql/README.md

```diff
@@ -16,12 +16,12 @@ Additionally, you can generate a native image and run the tests for this native
 mvn clean install -Dtest-containers -Dstart-containers -Dnative
 ```
 
-If you don't want to run PostgreSQL as a Docker container, you can start your own PostgreSQL server. It needs to listen on the default port and have a database called `hibernate_orm_test` accessible to the user `hibernate_orm_test` with the password `hibernate_orm_test`.
+If you don't want to run PostgreSQL as a Dev Service, you can start your own PostgreSQL server. It needs to listen on the default port and have a database called `hibernate_orm_test` accessible to the user `hibernate_orm_test` with the password `hibernate_orm_test`.
 
 You can then run the tests as follows (either with `-Dnative` or not):
 
 ```
-mvn clean install -Dtest-containers
+mvn clean install -Dtest-containers -Dpostgres.url=jdbc:postgresql://...
```

Comment:
I missed this in your other PRs... but I think it's incorrect? `postgres.url` is no longer mentioned in the configuration of this integration test, so I don't think setting it will do anything.

In order for this to be true, I think you'd need to keep the line `quarkus.datasource.jdbc.url=${postgres.url}` in `application.properties`. And set the default value of `postgres.url` to an empty string in `pom.xml`, so that by default the configuration will look like `quarkus.datasource.jdbc.url=` (no value), which means dev services will kick in.